site/book/01-getting-started/02-quickstart.md

```diff
@@ -80,6 +92,12 @@ This adds some metadata to the `Kptfile` required to keep track of changes made
 cluster. For example, if a resource is deleted from the package in the future, it will be pruned
 from the cluster.
 
+Create the namespace with name `my-space`
```

Comment:
Why can't have a Namespace resource declared in the package? That's the declarative way to do this and doesn't require bringing kubectl into the flow which is added complexity.

Reply:
It is a chicken-egg problem which is not solved yet. We need inventory object/RG to deploy package. We need namespace to deploy inventory object/RG to cluster. Creating namespace is a bootstrapping problem in kpt live for now.

CC @mortent @seans3

Reply:
I thought this was solved. If not, I rather not have `set-namespace` function until that's resolved.